presto-main/src/main/java/com/facebook/presto/operator/Driver.java

```diff
@@ -362,6 +366,18 @@ private void processNewSources()
                 return NOT_BLOCKED;
             }
 
+            // Some operators (LookupJoinOperator and HashBuildOperator) are broken and requires finish to be called continuously
```

Comment:
`s/requires/require`

also, comment says about "LookupJoinOperator and HashBuildOperator", but code below does only `instanceof LookupJoinOperator`?

also, i see no way to fix `HashBuildOperator` so that it doesn't require finish to be called multiple times. Should we consider this operator eternally "broken"? Maybe consider removing "broken" from the comment?